src/app/components/clients/clients.component.ts

```diff
@@ -16,28 +18,50 @@ import {SnackBarService} from '../../services/snack-bar/snack-bar-service';
 
 export class ClientsComponent implements OnInit {
   allClients: Client[];
-  errorMessage: string;
-  Client = <Client> {};
-
+  realm: Realm;
+  Client: Client;
+  private subscription: Subscription;
   displayedColumns: string[] = ['name'];
+
   form = new FormGroup({
     clientName: new FormControl('', Validators.required),
   });
 
   constructor(private changeDetectorRefs: ChangeDetectorRef,
               private service: ClientService,
               private snackBar: SnackBarService,
-              public dialog: MatDialog) {
+              private dialog: MatDialog,
+              private clientService: ClientService) {
```

Comment:
de ce am 2x ClientService?

Reply:
Cand am modificat numele de la servicii am adaugat in plus un ClientService din greseala.